styleguide/STYLE.md

```diff
@@ -171,7 +171,7 @@ Our docs website is a custom build. This section gives some guidance on working
 ### Adding and naming new documentation pages
 To add a new documentation page, create it as a *.md.erb file. Give it a lowercase name, separate words using underscores.
 To add the new page to the documentation sidebar on https://buildkite.com/docs, add the corresponding entry to
-`app/views/layouts/application.html.erb` with a description (e.g. `"G Cloud Identity", 'integrations/sso/g-cloud-identity'` ).
+`app\views\layouts\_sidebar.html.erb` with a description (e.g. `"G Cloud Identity", 'integrations/sso/g-cloud-identity'` ).
```

Comment:
I think we use `/` for paths in the rest of the docs, let's stick to that.

Reply:
Oh, this is a weird serial typo. Fixing.